vscode-dotnet-runtime-extension/package.json

```diff
@@ -73,7 +73,16 @@
 				},
 				"dotnetAcquisitionExtension.existingDotnetPath": {
 					"type": "array",
-					"description": "File Path to an existing installation of .NET. Used for both the .NET SDK and .NET Runtime.",
+					"description": "File Path to an existing installation of .NET to be used by one extension. Used for both the .NET SDK and .NET Runtime. Restart VS Code to apply changes.",
+					"examples": [
+						"C:\\Program Files\\dotnet\\dotnet.exe",
+						"/usr/local/share/dotnet/dotnet",
+						"/usr/lib/dotnet/dotnet"
+					]
+				},
+				"dotnetAcquisitionExtension.sharedExistingDotnetPath": {
+					"type": "string",
```

Comment:
What's the difference between this and the existing `dotnetAcquisitionExtension.existingDotnetPath` setting?

Reply:
`existingDotnetPath` is a setting that has existed for about 5 years. With that setting, you need to specify a path and an extension id. The path is only used for that extension. If you want MAUI, C#, C#DK to all use the same path, you need to set it for all 3 extension ids individually. Many users dont realize this.

`sharedExistingDotnetPath` does not require the extension id. It's only a path string, and it will apply to all extensions at once that call us.